Update React Native README.md (#52)
mParticle/react-native-mparticle: README.md

}
```

> > **Warning:** Don't log events in your `Application.onCreate()`. Android may instantiate your `Application` class in the background without your knowledge, eventwhile the user isn't even using their device, and lead to unexpected results. 

Reviewer: ```suggestion
> **Warning:** Don't log events in your `Application.onCreate()`. Android may instantiate your `Application` class in the background without your knowledge, including when the user isn't using their device, and lead to unexpected results. 
```

MParticle.logCommerceEvent(event)
```

To log screen events:

```js
MParticle.logScreenEvent('Test screen', { 'Test key': 'Test value' })
```

## User

> Get the current user in order to apply and remove attributes, tags, etc.
> 
> Use `Identify` or `currentUser` methods to retrieve the `userID` for these calls

Reviewer: Seems like the react api looks different from the Flutter one here. This should be:

```suggestion
To set, remove, and get user details, call the `User` or `Identity` methods as follows:
```
